share/completions/rclone.fish

```diff
@@ -0,0 +1 @@
+command rclone completion fish 2>/dev/null | source
```

Comment:
We don't typically add `command` here.

The completions are sourced when something called `rclone` exists, this could also be a function. If that happens, this would error out.

Please remove it.